Plugin/TUFX/Settings/TUFXGameSettings.cs

```diff
@@ -22,26 +22,37 @@ public class TUFXGameSettings : GameParameters.CustomParameterNode
 
         public override bool HasPresets => false;
 
+        public TUFXGameSettings()
+        {
+            var defaults = TexturesUnlimitedFXLoader.defaultConfiguration;
+            FlightSceneProfile = defaults.FlightSceneProfile;
+            IVAProfile = defaults.IVAProfile;
+            MapSceneProfile = defaults.MapSceneProfile;
+            EditorSceneProfile = defaults.EditorSceneProfile;
+            SpaceCenterSceneProfile = defaults.SpaceCenterSceneProfile;
+            TrackingStationProfile = defaults.TrackingStationProfile;
+        }
+
         [GameParameters.CustomStringParameterUI("TUFX Profiles - Must be selected through in-game profile editor.", gameMode = GameParameters.GameMode.ANY, lines = 3, toolTip = "Profiles only shown here to enable per-save-game persistence.")]
         public string WarningLabel = string.Empty;
 
         [GameParameters.CustomStringParameterUI("Flight Scene Profile: ", gameMode = GameParameters.GameMode.ANY, lines = 1, toolTip = "Active Profile in the Flight Scene")]
-        public string FlightSceneProfile = "Default-Flight";
+        public string FlightSceneProfile = string.Empty;
```

Comment:
These strings could be left uninitialized if they're all set in the constructor right?